tensorflow/compiler/xrt/xrt_device.h

```diff
@@ -50,19 +52,32 @@ class XRTGenericDeviceAccessor {
     xla::LocalClient* client() const { return client_; }
     xla::Backend* backend() { return client_->mutable_backend(); }
     int device_ordinal() const { return ordinal_; }
+    // XRTCompileOp needs to use a memory allocator constructed with stream.
+    se::DeviceMemoryAllocator* GetMemoryAllocator(OpKernelContext* ctx);
```

Comment:
hmm, seems like `XRTTpuDeviceAccessor` should also implement `GetMemoryAllocator` since stateOP implementation is shared between tpu and cpu/gpu. I think it OK to always return `client_->mutable_backend()->memory_allocator()`

Reply:
Done.